src/account_manager.rs

```diff
@@ -387,7 +392,7 @@ impl Clone for AccountManager {
 
 impl Drop for AccountManager {
     fn drop(&mut self) {
-        self.stop_background_sync();
+        self.stop_background_sync().unwrap();
```

Comment:
It doesn't really feel like the right place to do that to me. I would expect the user to have to call a shutdown mechanism themselves and have the opportunity to act on the error instead of panicking.

Reply:
And what would happen if the user doesn't call it before dropping it?

Reply:
You tell me, I don't know what `stop_background_sync` does. But if the user doesn't call it then they are misusing the API.

Reply:
It would panic then in the background syncing thread which we want to stop here so that doesn't happen